packages/frontend/api/message.ts

```diff
@@ -0,0 +1,10 @@
+import { MessageInterface } from '../interfaces/message'
+import axios from 'axios'
+
+const messageInstance = axios.create({
+  baseURL: `${process.env.NEXT_PUBLIC_SERVER_URL}/message`,
+})
+
+export async function createMessage(data: MessageInterface) {
+  await messageInstance.post('', data)
```

Comment:
I think you should move the specific path to the specific function.
```suggestion
  baseURL: process.env.NEXT_PUBLIC_SERVER_URL,
})

export async function createMessage(data: MessageInterface) {
  await messageInstance.post('/message', data)
```